Mage.Sets/src/mage/sets/MurdersAtKarlovManor.java

```diff
@@ -484,15 +484,18 @@ private MurdersAtKarlovManor() {
         cards.add(new SetCardInfo("Yarus, Roar of the Old Gods", 375, Rarity.RARE, mage.cards.y.YarusRoarOfTheOldGods.class, NON_FULL_USE_VARIOUS));
     }
 
-    // @Override
-    // protected void generateBoosterMap() {
-        // super.generateBoosterMap();
-        // CardRepository
-                // .instance
-                // .findCards(new CardCriteria().setCodes("SPG"))
-                // .stream()
-                // .forEach(cardInfo -> inBoosterMap.put("SPG_" + cardInfo.getCardNumber(), cardInfo));
-    // }
+    @Override
+    protected void generateBoosterMap() {
+        super.generateBoosterMap();
+        for( let cn = 19 ; cn < 29 ; cn++ ){
+            inBoosterMap.put("SPG_" + cn, CardRepository.instance.findCard("SPG", "" + cn));
```

Comment:
Must check findCard result and save only non nullable values. inBoosterMap allow only good objects. Also on null values you can raise error about unknown card (it will help to find typo or non-implemented cards in booster settings).

![IMG_0900](https://github.com/user-attachments/assets/1c917475-545c-4053-b777-ded2957f40e1)